frontend/app/assets/javascripts/spree/frontend/checkout.js

```diff
@@ -59,3 +59,17 @@ Spree.ready(function () {
 
   return Spree.Checkout
 })
+
+$('.js-remove-credit-card').click(function() {
```

Comment:
@damianlegawiec Do we want some credit card remove confirmation popup/modal here?

Reply:
@szymoniwacz standard `alert` or `data-confirm` will work